docs/preparation_for_tests.md

```diff
@@ -0,0 +1,136 @@
+# Preparation for tests
+
+This document describes step by step how to prepare the environment to run
+Fobnail and provision it.
+
+## Requirements
+
+Files that you should have before starting the procedure:
+
+* root.crt
+* provision.sh
+* ca2.crt
+* ca2.priv
+* ca2.srl
+* cc_cbor.py
+* chain.pem
+* leaf.ext
+
+## The procedure of running the CoAP server
+
+1. [Install docker](https://docs.docker.com/engine/install/) using the guide
+   for Linux distribution present on your computer.
+1. Add your user to the docker group: `sudo usermod -aG docker $USER`. Then log
+    out of the desktop session then log in again. This step is one-time only.
+1. Prepare the Fobnail environment by opening the terminal and running the
+    following commands:
+
+    ```bash
+    git clone git@github.com:fobnail/fobnail.git
+    cd fobnail/
+    git checkout token_provisioning
+    git submodule update --init --recursive
+    ```
+
+1. Put file `run-fobnail-sdk.sh` from `fobnail-sdk` repository in PATH by
+    running the following commands in the new terminal:
+
+    ```bash
+    git clone git@github.com:fobnail/fobnail-sdk.git
+    cd fobnail-sdk/
+    mv run-fobnail-sdk.sh ~/.local/bin/
+    source ~/.bashrc
+    ```
+
+1. [Configure Network](https://fobnail.3mdeb.com/environment/#networking-setup),
+    for a fast but temporary solution, you can only use the first 3 commands.
+1. Put the `root.crt` file in the main `fobnail` dictionary.
+1. Prepare some variables by running the following commands:
```

Comment:
This step may be removed, please put exports before `./build.sh --run` in step 8

Reply:
`exports` transferred to another step